### PR TITLE
add meta tags for Twitter

### PR DESCRIPTION
**Pull request by lunson**
Allows logo preview when ome.org is linked in a tweet

### CODE REVIEW

**Comment by hflynn** [comment]
I can't seem to test this using the snoopy site so we might have to just merge it and see if it works!

**Comment by hflynn** [comment]
Sadly implementing part of this on the blog doesn't seem to have worked:

blog_twitter_preview

**Comment by hflynn** [comment]
Ah, I shouldn't have removed the 'title' section for the blog - https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/summary tells me it is required.

So no reason why this shouldn't work here.

**Comment by hflynn** [comment]
Not having any joy getting this working on the blog this afternoon but I was maybe too impatient and didn't give the merged code time to filter through to the live site. I'll retest the blog update tomorrow and if it works then, we'll know this should work too.

File: _includes/head.html [hunk 1]
```diff
@@ -10,6 +10,10 @@
         <meta property="og:url" content="{{ site.baseurl }}" />
         <meta property="og:title" content="The Open Microscopy Environment" />
         <meta property="og:description" content="A consortium of universities, research labs, industry and developers producing open-source software and format standards for microscopy data." />
+        <meta name="twitter:title" content="The Open Microscopy Environment">
+        <meta name="twitter:description" content="A consortium of universities, research labs, industry and developers producing open-source software and format standards for microscopy data.">
+        <meta name="twitter:image" content="{{ site.baseurl }}">
+        <meta name="twitter:card" content="{{ site.baseurl }}/img/logos/ome-logo_fb.png">
```
Comment (hflynn) [inline _includes/head.html]: This image location needs to be under "image" and the content of the "card" has to be "summary"

Not sure if the order matters too, but https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/summary has the "card" item first

**Comment by hflynn** [comment]
Finally got this to work for the blog on https://github.com/ome/blog/pull/55 - you need to use the full URL for the image too

File: _includes/head.html (hunk 1 above)
Comment (hflynn) [reply _includes/head.html]: Also need to full URL not the site.baseurl code

Comment (lunson) [reply _includes/head.html]: Okay, I've pushed changes in the next commit.

**Comment by hflynn** [comment]
Changes look good, this is now the same as what's working on the blog. I'm not entirely sure what this logo will look like cropped to a square but maybe we can just change it later if it's not pretty.

@joshmoore ok to merge for deployment today with the 5.4.2 retraction?

**Comment by joshmoore** [comment]
WFM

**Comment by hflynn** [comment]
This is what it looks like:

website_preview

Don't know if we are bothered by the cropped logo or not. I do wonder if we should remove the description so that the field is autofilled by text from the actual page though.

**Comment by lunson** [comment]
I'll work on a square logo and leave the description blank. I'll open a separate PR.

**Comment by hflynn** [comment]
Thanks @lunson

**Comment by hflynn** [comment]
Also, a heads-up, it only works for *https* - don't know if there is anything we can do about that

**Comment by lunson** [comment]
@hflynn Do you mean use https when linking to the logo for the image meta tag?

**Comment by hflynn** [comment]
Sorry, I mean the image didn't work when I forgot we default to https and posted a link to http://openmicroscopy.org instead.

**Comment by hflynn** [comment]
I should really remember to use https though so not a big deal!

**Pull request merged**